app/controllers/merchants_controller.rb

```diff
@@ -29,4 +29,56 @@ def admin_or_merchant
   def not_today_satan
     render_404 unless current_merchant?
   end
+
+  def new
+    @item = Item.new
+  end
+
+  def create    
+    thumbnail = params[:item][:thumbnail]
+
+    File.open(Rails.root.join('app', 'assets', 'images', thumbnail.original_filename), 'wb') do |file|
+      file.write(thumbnail.read)
+    end
+    
+    @merchant = current_user
+    @item = @merchant.items.create(item_params)
+    redirect_to dashboard_items_path
+    # if @item.save
+    #   flash[:success] = "Item added!"
```

Comment:
We'll eventually need to delete the commented out portion, but I'm fine with this for now.